pom.xml

```diff
@@ -45,8 +45,20 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>2.0.1.Final</version>
```

Comment:
it seems like validation doesn't work, check please

Reply:
removed and changed to Spring boot starter validation